plutus-ledger-api/test/Spec/CostModelParams.hs

```diff
@@ -32,8 +32,8 @@ tests =
             166 @=? length V1.costModelParamsForTesting
             175 @=? length v2_ParamNames
             175 @=? length V2.costModelParamsForTesting
-            223 @=? length v3_ParamNames
-            223 @=? length V3.costModelParamsForTesting
+            233 @=? length v3_ParamNames
+            233 @=? length V3.costModelParamsForTesting
```

Comment:
Should be golden tests, I think.

Reply:
> Should be golden tests, I think.

Hmmm.  I just extended what was there already.  @bezirg ?